crates/api_models/src/admin.rs

```diff
@@ -347,6 +377,70 @@ pub struct MerchantConnector {
     pub metadata: Option<pii::SecretSerdeValue>,
 }
 
+/// Create a new Merchant Connector for the merchant account. The connector could be a payment processor / facilitator / acquirer or specialized services like Fraud / Accounting etc."
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct MerchantConnectorUpdate {
+    /// Type of the Connector for the financial use case. Could range from Payments to Accounting to Banking.
+    #[schema(value_type = ConnectorType, example = "payment_processor")]
+    pub connector_type: api_enums::ConnectorType,
+    /// Name of the Connector
+    #[schema(example = "stripe")]
+    pub connector_name: String,
+
+    /// Business Sub label of the merchant
+    #[schema(example = "chase")]
+    pub business_sub_label: Option<String>,
```

Comment:
We cannot let them change these fields when updating connector information since the connector label needs to be immutable. I'd suggest either removing them or skipping deserializing these.

Reply:
This field is removed from the updatable fields.

Reply:
Shouldn't we also remove the connector name?

Reply:
Oh right, that would also affect connector label. Yes connector name should also not be allowed to update.